lib/graphql/schema/list.rb

```diff
@@ -20,9 +20,11 @@ def list?
         true
       end
 
-      def to_type_signature
-        "[#{@of_type.to_type_signature}]"
+      def graphql_name
+        "[#{@of_type.graphql_name}]"
       end
+
+      alias_method :to_type_signature, :graphql_name
```

Comment:
I tried putting this in `GraphQL::Schema::Wrapper` to remove code duplication but it would alias to the `graphql_name` in `GraphQL::Schema::Wrapper` and as a result raise a `NotImplementedError`. It looks like this is  how `alias_method` works, just kind of sucks. I might be wrong though, do you know otherwise?